src/servers/http_server.cc

```diff
@@ -78,7 +78,13 @@ HTTPServer::Start()
     evhtp_enable_flag(htp_, EVHTP_FLAG_ENABLE_NODELAY);
     evhtp_set_gencb(htp_, HTTPServer::Dispatch, this);
     evhtp_use_threads_wexit(htp_, NULL, NULL, thread_cnt_, NULL);
-    evhtp_bind_socket(htp_, "0.0.0.0", port_, 1024);
+    if (evhtp_bind_socket(htp_, "0.0.0.0", port_, 1024) != 0) {
```

Comment:
Only some of the `evhtp` functions return an error? Do we need to do error checking for other functions too?

Reply:
Thanks for checking David!

> Do we want to make a ticket for all the changes? 

Yes, I think we need to have a separate ticket to check the error of evhtp calls that do return an error.

Reply:
Any time! Created the ticket here: https://jirasw.nvidia.com/browse/DLIS-3580